src/main/index.js

```diff
@@ -47,7 +49,81 @@ function runApp() {
           browserWindow.webContents.selectAll()
         }
       }
-    ]
+    ],
+    // only show the copy link entry for external links and the /playlist, /channel and /watch in-app URLs
+    // the /playlist, /channel and /watch in-app URLs get transformed to their equivalent YouTube URLs
+    append: (defaultActions, parameters, browserWindow) => {
+      let visible = false
+      const isInAppUrl = parameters.linkURL.split('#')[0] === browserWindow.webContents.getURL().split('#')[0]
+
+      if (parameters.linkURL.length > 0) {
+        if (isInAppUrl) {
+          const path = parameters.linkURL.split('#')[1]
+
+          if (path) {
+            visible = ['/playlist', '/channel', '/watch'].some(p => path.startsWith(p))
+          }
+        } else {
+          visible = true
+        }
+      }
+
+      return [{
+        label: 'Copy Lin&k',
```

Comment:
I'm guessing the & is a typo?

Reply:
No, it makes that context menu entry activatable by clicking on it or pressing the `k` key while the context menu is open. As electron-context-menu does the same (https://github.com/sindresorhus/electron-context-menu/blob/main/index.js#L167) I thought it would be a good idea to copy it, so that the behaviour is the same (I should probably add it for the Select All option as well).

Reply:
Ahhh cool, didnt know that

Reply:
Please add code comment for it :S